src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -562,12 +573,16 @@ public Map<String, Long> createSnapshotParquetFiles(
               datasetDataSourceName,
               snapshotDataSourceName,
               datasetFlightId,
-              columns);
+              columns,
+              isGlobalFileIds);
+
+      logger.info("((((" + query);
```

Comment:
Yeah, though I'm going to leave the logging (we do it for BQ queries as well).  It can be super useful for debugging

Reply:
I'll remove this particular statement

Reply:
Definitely useful for debugging.
I'd still urge us to remove it before merging for a few reasons listed below (BUT, I kinda hope y'all can prove my concerns wrong b/c I do think it would be helpful):

- This would print out the secret that's used to create the scoped credentials. While this does expire, I would still think this is a security risk to have this printed in this less-secure location.
- This is going to produce a lot of logs($$$), especially as we move to the larger by query snapshot creation where we have to walk through each table 
- There is an alternative --  There is a "monitor" feature in synapse studio where you can search through recent queries
<img width="1297" alt="image" src="https://user-images.githubusercontent.com/13254229/203591514-34ef7c03-8588-47d9-ab7a-1baeb9b496e8.png">

Reply:
Note: going to change to use debug logging for this